client/my-sites/migrate/step-import-or-migrate.jsx

```diff
@@ -64,11 +65,18 @@ class StepImportOrMigrate extends Component {
 		return planSlug && planHasFeature( planSlug, FEATURE_UPLOAD_THEMES_PLUGINS );
 	};
 
+	installJetpack = () => {
+		// recordTracksEvent( 'calypso_site_importer_install_jetpack' );
```

Comment:
Was this commented out by accident?

Reply:
Thanks @bogiii . I'm commenting it out for now because I don't want to introduce a new/duplicate event in until we figure out what's the best way for us to track the in-product page migration flow. So I ended up focus the functionality in the PR and plan to add the event in once we figure out what's the best to do.